modules/redis/charts.go

```diff
@@ -0,0 +1,190 @@
+package redis
+
+import "github.com/netdata/go-orchestrator/module"
+
+type (
+	// Charts is an alias for module.Charts
+	Charts = module.Charts
+	// Dims is an alias for module.Dims
+	Dims = module.Dims
+)
+
+const (
+	keysRedisChartId = "keys_redis"
+)
+
+var charts = Charts{
+	{
+		ID:    "operations",
+		Title: "Operations",
+		Units: "operations/s",
+		Fam:   "operations",
+		Ctx:   "redis.operations",
+		Type:  module.Line,
+		Dims: Dims{
+			{
+				ID:   "total_commands_processed",
+				Name: "commands",
+				Algo: module.Incremental,
+			},
+			{ID: "instantaneous_ops_per_sec", Name: "operations"},
+		},
+	},
+	{
+		ID:    "hit_rate",
+		Title: "Hit rate",
+		Units: "percentage",
+		Fam:   "hits",
+		Ctx:   "redis.hit_rate",
+		Type:  module.Line,
+		Dims: Dims{
+			{ID: "hit_rate", Name: "rate"},
+		},
+	},
+	{
+		ID:    "memory",
+		Title: "Memory utilization",
+		Units: "KiB",
+		Fam:   "memory",
+		Ctx:   "redis.memory",
+		Type:  module.Line,
+		Dims: Dims{
+			{
+				ID:   "used_memory",
+				Name: "total",
+				Div:  1024,
+			},
```

Comment:
lets have one line dimension, i would prefer to have them consistent across all charts

Reply:
Sure, I can do that. Gotta be honest though, first time since I started working with Go that I see a comment in a code review about formatting. It's fine it's just that I usually prefer to see more constructive comments and I thought I'd point that out.

I hope you don't mind the criticism :slightly_smiling_face:

Reply:
no, all is good